docker/run_on_entry.sh

```diff
@@ -39,23 +32,34 @@ type = cloud" >> /etc/apel/clouddb.cfg
 
 echo "
 ALLOWED_FOR_GET=$ALLOWED_FOR_GET
-SERVER_IAM_ID=$SERVER_IAM_ID
-SERVER_IAM_SECRET=$SERVER_IAM_SECRET
+SERVER_IAM_ID=\"$SERVER_IAM_ID\"
+SERVER_IAM_SECRET=\"$SERVER_IAM_SECRET\"
+
```

Comment:
Yes, not having it defined in the apel_rest/settings.py file causes the django started to fail to even start and hence the unit tests all fail. 

Whereas the other variables can be set with the relevant unit tests.